docs/source/users/options/minimizer_option.rst

```diff
@@ -343,14 +363,16 @@ The `matlab_stats` minimizer is set as follows:
 
 	   
 Minuit (``minuit``)
--------------------
+===================
+
+CERN developed the `Minuit <http://seal.web.cern.ch/seal/snapshot/work-packages/mathlibs/minuit/>`__ package to find the minimum value of a multi-parameter function, and also to compute the uncertainties.
+We interface via the python interface `iminuit <https://iminuit.readthedocs.io>`__ with support for the 2.x series. 
 
-CERN developed the `Minuit <http://seal.web.cern.ch/seal/snapshot/work-packages/mathlibs/minuit/>`_ package to find the minimum value of a multi-parameter function, and also to compute the uncertainties.
-We interface via the python interface `iminuit <https://iminuit.readthedocs.io>`_ with support for the 2.x series. 
+* `Minuit's MIGRAD <https://root.cern.ch/root/htmldoc/guides/minuit2/Minuit2.pdf>`__ (:code:`minuit`)
 
-* `Minuit's MIGRAD <https://root.cern.ch/root/htmldoc/guides/minuit2/Minuit2.pdf>`_ (:code:`minuit`)
+**Links** `Github - iminuit <https://github.com/scikit-hep/iminuit>`__
 
-**Links** `Github - iminuit <https://github.com/scikit-hep/iminuit>`_
+**Licence** iminuit is released under the `MIT licence <https://github.com/scikit-hep/iminuit/blob/develop/LICENSE>`__, while Minuit is `LGPL v2 <http://seal.web.cern.ch/seal/main/license.html>`__ .
```

Comment:
I get a forbidden message when I try to follow this Minuit link.

Reply:
Updated the seal links to root to fix this